docs/authorization-configuration.md

```diff
@@ -74,6 +77,39 @@ auth: {
 - **response_type** defaults to the **id_token**. Any other parameter that is added to oAuthData is also added to the authorization payload.
 - **nonceFn** provides a function that returns a string in order to override the default **nonce**.
 - **logoutFn** provides the function to override the **logoutUrl** functionality for a custom logout. It needs to execute the **logoutCallback()** function after logout.
+- **userInfoFn** provides a function to get user information. It returns a promise of an user info object which contains user name and/or email to display it in the profile dropdown.
+
+
+### Custom Authentification Provider
```

Comment:
I'm wondering, should it be `authentification` or `authentication`? I was researching similar providers and it was `authentication` 🤔  but I might be wrong :) However, it is used to authenticate the user right?

Reply:
You are totally right :)